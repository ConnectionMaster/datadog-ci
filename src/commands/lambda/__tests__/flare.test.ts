import {createMockContext, makeCli} from './fixtures'

describe('lambda flare', () => {
  it('prints non-dry-run header', async () => {
    const cli = makeCli()
    const context = createMockContext()
    const code = await cli.run(['lambda', 'flare'], context as any)
    const output = context.stdout.toString()
    expect(code).toBe(0)
    expect(output).toMatchSnapshot()
  })

  it('prints dry-run header', async () => {
    const cli = makeCli()
    const context = createMockContext()
    const code = await cli.run(['lambda', 'flare', '-d'], context as any)
    const output = context.stdout.toString()
    expect(code).toBe(0)
    expect(output).toMatchSnapshot()
  })
})
