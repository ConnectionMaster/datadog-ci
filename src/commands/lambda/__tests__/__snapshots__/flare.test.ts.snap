// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lambda flare prints dry-run header 1`] = `
"
[Dry Run] 🐶 Generating Lambda flare to send your configuration to Datadog
"
`;

exports[`lambda flare prints non-dry-run header 1`] = `
"
🐶 Generating Lambda flare to send your configuration to Datadog
"
`;
